orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -102,19 +103,41 @@ public TaskResult execute(@Nonnull Stage stage) {
     StageData stageData = stage.mapTo(StageData.class);
     Map<String, Object> cluster;
 
+    AtomicReference<Moniker> moniker = new AtomicReference<>(stageData.moniker);
```

Comment:
Why `AtomicReference`?

Reply:
Only because it's also being referenced in a lambda that expects it to be final.